lesson3/code-tasks/task-5.js

```diff
@@ -0,0 +1,28 @@
+const path = '../users.json';
+
+const loadUsers = (path) => {
+  return fetch(path)
+    .then(response => {
+      if (!response.ok) {
+        throw new Error('Network response was not ok.');
+      }
+
+      return response.json();
+    })
+    .then(users => {
+      let originalUsers = users
+      originalUsers.forEach(element => {
+        if(element.salary && console.log(element.salary));
+      });
+
+      console.log('================================FILTERED===================================');
+      
+      let filteredUsers = users.filter(user => user.salary > 1000)
+      filteredUsers.forEach(element => {
+        if(element.salary && console.log(element.salary));
```

Comment:
`if(element.salary && console.log(element.salary))` should be rewritten to `element.salary && console.log(element.salary)` without a `if`.

And still it might be better to check if user has a `salary` property in the filter function.